src/test/java/rizzler/TaskListTest.java

```diff
@@ -0,0 +1,74 @@
+package rizzler;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.AfterEach;
+import static org.junit.jupiter.api.Assertions.fail;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.util.ArrayList;
+
+public class TaskListTest {
+    private static final ByteArrayOutputStream testOut = new ByteArrayOutputStream();
+    private static final PrintStream originalOut = System.out;
+
+    @BeforeEach
+    public void setStreams() {
+        System.setOut(new PrintStream(testOut));
+    }
+
+    @AfterEach
+    public void restoreStreams() {
+        System.setOut(originalOut);
+    }
```

Comment:
I like the way you set up and clean up the test using https://github.com/beforeeach and @AfterEach annotations. Implementing advance knowledge from the course takes effort and you are doing it very well👍